### PR TITLE
Refactor aggregate to in_parallel

### PR DESCRIPTION
**Pull request by jontours**
@hillaryj @RonWilliams We need to do this across all our pipelines, sooner than later, to head off any compatibility issues with future releases of concourse.

### CODE REVIEW

File: ci/pipeline.yml [hunk 1]
```diff
@@ -3,14 +3,14 @@ jobs:
...
-  - aggregate:
+  - in_parallel:
     - get: ca-cert-store
     - get: bosh-deployment
       trigger: true
     - get: bosh-config
       resource: bosh-config-development
       trigger: true
-    - get: bosh-stemcell
+    - get: bosh-stemcell-xenial
```
Comment (hillaryj) [inline ci/pipeline.yml]: What does switching from `bosh-stemcell` to `bosh-stemcell-xenial` do?

File: ci/pipeline.yml [hunk 2]
```diff
@@ -406,6 +402,22 @@ jobs:
       - terraform-yaml/state.yml
       - common-tooling/tooling-bosh.yml
       ops_files:
+      - bosh-deployment/uaa.yml
```
Comment (hillaryj) [inline ci/pipeline.yml]: What do these `ops_files` add?

**Comment by hillaryj** [comment]
Is there a way to test these changes before merging, or has that already been done?

File: ci/pipeline.yml (hunk 1 above)
Comment (mxplusb) [reply ci/pipeline.yml]: The default resource points to trusty, this makes it clear we leverage xenial stemcells.

File: ci/pipeline.yml (hunk 2 above)
Comment (mxplusb) [reply ci/pipeline.yml]: It's a [yaml merge syntax](https://bosh.io/docs/cli-ops-files/), and this is the way that functionality is exposed in the bosh deployment resource in Concourse.

**Comment by mxplusb** [comment]
> Is there a way to test these changes before merging, or has that already been done?

Not without a non-trivial amount of work, which is one of the, imo, fundamental limitations of Concourse. However, you can run `fly validate-pipeline` to at least validate it's syntactically correct.

**Pull request merged**